azure-pipelines.yml

```diff
@@ -8,6 +8,10 @@ jobs:
     variables:
       pythonVersion: "3.9"
       cmakeBuildType: Release
+      gudhiCmakeOptions: -DWITH_GUDHI_EXAMPLE=ON -DWITH_GUDHI_TEST=ON -DWITH_GUDHI_UTILITIES=ON -DWITH_GUDHI_PYTHON=ON -DWITH_GUDHI_REMOTE_TEST=ON
+      # On this VM, 2 versions of python are installed. Default Python_FIND_FRAMEWORK is FIRST, which means asks frameworks first
+      # LAST means consult frameworks in last resort, use the standard libraries or headers first
```

Comment:
I would expect it uses PATH, not "standard libraries or headers", since it is looking for a python exe?